### PR TITLE
Hint Pane/Button

### PR DESCRIPTION
**Pull request by vivianliu0**
Added hint button to the bottom right of the screen, which opens up a text pane displaying the hints for a scene.
The hint button has two states, "New Hints" and "Hints". Initially, the state is set to "New Hints", and after the hint button has been clicked, the button changes to the "Hints" state.

https://www.loom.com/share/e1158059d7d740d09171b55c03f1cbd9

### CODE REVIEW

**Comment by aaronabraham311** [comment]
Looks great! Nice work

**Comment by ahmedhamodi** [comment]
Does it only change to "Hints" once all hints are viewed, or when the popup is opened for the first time?

File: samples/hint_panes/components/hint_button.js [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+      blackboardData: {
+        componentType: "text_pane",
+        jsonData: {
+          text: [
+            "Hint 1: Have you checked the folded page in your textbook? There might be some useful info there...",
+            "Hint 2: Filler text",
+            "Hint 3: Filler text",
+          ],
+          currPosition: 0,
+        },
```
Comment (ahmedhamodi) [inline samples/hint_panes/components/hint_button.js:59]: @jaydhulia - you'll work on integrating this with the hints field in the schema right?

**Comment by vivianliu0** [comment]
> Does it only change to "Hints" once all hints are viewed, or when the popup is opened for the first time?
The button state is changed once the popup is opened for the first time

**Review by jaydhulia — COMMENTED** [review]
Looks great! Small comments, awesome work overall @vivianliu0 🎉

File: samples/hint_panes/components/hint_button.js (hunk 1 above)
Comment (jaydhulia) [reply samples/hint_panes/components/hint_button.js:59]: Yes, when we are moving this to the `app/public` folder, I will integrate this to use data from Postgres

File: samples/hint_panes/hint_panes.html [hunk 2]
```diff
@@ -0,0 +1,62 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8" />
+    <title>Hint Pane Demo</title>
+    <meta name="description" content="Hint pane demo" />
+    <script src="https://aframe.io/releases/1.2.0/aframe.min.js"></script>
```
[the remark this answers is not in the document]
Comment (vivianliu0) [reply samples/hint_panes/hint_panes.html]: Updated to 1.1.0, verified that it's still working

File: samples/hint_panes/hint_panes.html [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+        rotation="0 0 0"
+        fov="80"
+        camera
+        look-controls
+        wasd-controls
+      >
+        <a-entity
+          id="hint-button"
+          class="hint-new"
+          cursor="fuse: true; fuseTimeout: 500"
```
[the remark this answers is not in the document]
Comment (vivianliu0) [reply samples/hint_panes/hint_panes.html]: Oops this isn't needed/shouldn't be there, I was playing around with it and essentially a fuse based cursor triggers a click if the user gazes at an entity longer than the fuseTimeout. Removed in the latest commit

**Comment by ahmedhamodi** [comment]
@vivianliu0 can you update the branch to align with master? You can just click the "update branch" button here:
image

**Review by jaydhulia — APPROVED** [review]
Awesome, LGTM! Merge way 🚀

**Pull request merged**